packages/npm/@amazeelabs/graphql-directives/src/resolvers.ts

```diff
@@ -80,6 +92,7 @@ export function buildResolver(
             processDirectiveArguments(value, args, spec),
             context,
             info,
+            gatsbyHelpers
```

Comment:
I would merge the helper API into the `context`. Thats also where other Gatsby API's like `nodeModel` live. Then we don't have to deviate from the standard GraphQL resolver signature.

Reply:
done, the helpers are now merged into context.